cms/envs/help_tokens.ini

```diff
@@ -2,39 +2,39 @@
-export_library = course_author:course_components/libraries.html#export-a-library
-export_course = course_author:releasing_course/export_import_course.html#export-a-course
-welcome = course_author:getting_started/index.html
-login = course_author:getting_started/index.html
-register = course_author:getting_started/index.html
-content_libraries = course_author:course_components/libraries.html
-content_groups = course_author:course_features/cohorts/cohorted_courseware.html
-enrollment_tracks = course_author:course_features/diff_content/enroll_track_courseware.html
-group_configurations = course_author:course_features/content_experiments/content_experiments_configure.html#set-up-group-configurations-in-edx-studio
-container = course_author:developing_course/course_components.html#components-that-contain-other-components
-video = course_author:video/index.html
-certificates = course_author:set_up_course/studio_add_course_information/studio_creating_certificates.html
-content_highlights = course_author:developing_course/course_sections.html#set-course-section-highlights
-image_accessibility = course_author:accessibility/best_practices_course_content_dev.html#use-best-practices-for-describing-images
-social_sharing = course_author:developing_course/social_sharing.html
+default = course_author:quickstarts/build_a_course.html
+home = course_author:concepts/open_edx_platform/what_is_studio.html
+develop_course = course_author:how-tos/set_up_course/create_new_course.html
+outline = course_author:concepts/open_edx_platform/about_course_outline.html
+unit = course_author:concepts/open_edx_platform/about_course_units.html
+visibility = course_author:references/controlling_content_visibility.html
+updates = course_author:concepts/communication/about_course_updates_handouts.html
+pages = course_author:how-tos/course_development/manage_custom_page.html
+files = course_author:references/course_development/files_page.html
+textbooks = course_author:how-tos/course_development/manage_textbooks.html
+schedule = course_author:how-tos/set_up_course/set_course_schedule.html
+grading = course_author:how-tos/grading/set_grade_range.html
+team_course = course_author:references/course_development/course_team_roles.html
+team_library = course_author:how-tos/course_development/add_users_to_libraries.html
+advanced = course_author:quickstarts/build_a_course.html
+checklist = course_author:concepts/releasing-course/course_launch_checklist.html
+import_library = course_author:how-tos/course_development/export_import_library.html#import-a-legacy-library
+import_course = course_author:how-tos/releasing-course/import_course.html
+export_library = course_author:how-tos/course_development/export_import_library.html#export-a-legacy-library
+export_course = course_author:how-tos/releasing-course/export_course.html
+welcome = course_author:index.html
+login = course_author:index.html
+register = course_author:index.html
```

Comment:
Where are these used? I'm not sure the course_author:index is a very useful path to point to